examples/kubernetes/1.10/cilium-containerd-ds.yaml

```diff
@@ -175,7 +175,7 @@ spec:
               key: auto-direct-node-routes
               name: cilium-config
               optional: true
-        image: docker.io/cilium/cilium:v1.4.0
+        image: docker.io/cilium/cilium:1.4
```

Comment:
@eloycoto this should not have been changed - it should still be `v1.4.0`.

Reply:
is already fixed ;-)

Reply:
How is it fixed?

Reply:
Comment is outdated: 
https://github.com/cilium/cilium/pull/7108/files#diff-bb04b0a781aef2e330d7f84c05d4b78fR343